packages/react/src/components/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -111,18 +81,18 @@ describe('Breadcrumb', () => {
       { text: 'Test3', key: 'Test3', as: 'h1' },
     ];
 
-    wrapper = mount(<Breadcrumb items={items2} />);
+    const { container } = render(<Breadcrumb items={items2} />);
 
     // get the first child of each list item (the actual item content)
-    const renderedItems = wrapper.find('.ms-Breadcrumb-listItem > *:first-child');
+    const renderedItems = container.querySelectorAll('.ms-Breadcrumb-listItem > *:first-child');
```

Comment:
If possible it would be nice to switch to a more semantic way of getting the items, like if they have a particular role, or maybe getting by text. Same for other tests.

Reply:
Remove all usage of `querySelector` in favor of accessing hidden roles and using `firstElementChild`